.codacy.yaml

```diff
@@ -0,0 +1,10 @@
+exclude_paths:
+  - '.bundle/**'
+  - 'spec/**/*'
+  - 'benchmarks/**/*'
+  - '*.min.js'
+  - '**/tests/**'
+  - 'wiki/**'
+  - '.github/**'
+  - 'CODE_OF_CONDUCT.md'
+  - 'CONTRIBUTING.md'
```

Comment:
Are you sure all these exclusions are valid?

Reply:
No, just copied from EI back-end. But they look like they should work. Exclusion does not yet display since it is not merged in master, thus you may still se faults in for example CODE_OF_CONDUCT.md

Reply:
You don't have bundle, spec, benchmarks, min.js and wiki in the project why then exclude them?

Reply:
Guess they are default since that are in EI front-end, back-end and gerrit-plugin. If this bothers you we should update in all repositories.

Reply:
No, we don't have any default, please check https://github.com/eiffel-community/eiffel-gerrit-plugin/blob/master/.codacy.yaml.

I don't like excluding things that are not part of the repo

Reply:
Already fixed.
https://github.com/eiffel-community/eiffel-intelligence-frontend/blob/master/.codacy.yaml and https://github.com/eiffel-community/eiffel-intelligence/blob/master/.codacy.yaml needs to be updated as well.